src/test/java/com/google/devtools/build/lib/buildtool/CustomRealFilesystemBuildIntegrationTest.java

```diff
@@ -353,11 +354,10 @@ public void ioExceptionReadingBuildFileForDiscoveredInput() throws Exception {
     buildTarget("//hello:hello");
     BuildFailedException e =
         assertThrows(BuildFailedException.class, () -> buildTarget("//hello:hello"));
-    MoreAsserts.assertContainsEventRegex(
-        events.collector(),
+    MoreAsserts.assertContainsEvent(events.collector(), Pattern.compile(
```

Comment:
pretty sure our internal linter is going to complain about compiling a constant string pattern inline. Not worth refactoring, just FYI.